bundles/org.openhab.binding.panasonictv/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="panasonictv" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>Panasonic TV Binding</name>
+	<description>This is the binding for Panasonic TV. Binding should support all Panasonic Viera TV models</description>
+	<author>Prakashbabu Sidaraddi</author>
```

Comment:
The author tag is deprecated and should therefore be removed. See openhab/openhab-core#1844.

Reply:
This needs to be adapted to the new `addon.xml`, the PR to adapt the developer docs is not merged yet, so if you need more details, the corresponding issue is here: https://github.com/openhab/openhab-core/issues/2058
But you can also update your branch and look at another binding's `addon.xml` and adapt it to this initial contribution.